frontend/src/components/utils/ExecutionReceiptStatus.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReceiptExecutionStatus } from "../../libraries/explorer-wamp/receipts";
+
+const EXECUTION_RECEIPT_STATUSES: Record<ReceiptExecutionStatus, string> = {
+  Unknown: "Unknown",
+  Failure: "Failed",
+  SuccessValue: "Succeeded",
+  SuccessReceiptId: "SuccessReceiptId",
```

Comment:
Whenever we observe `SuccessReceiptId`, we should display something like "Status is deferred to receipt <receipt-id>"